.github/workflows/build.yml

```diff
@@ -72,9 +72,9 @@ jobs:
           AZURECR_PAT: ${{ secrets.AZURECR_PAT }}
           GOOGLECR_KEYS: ${{ secrets.GOOGLECR_KEYS }}
 
-      - name: Cleanup build workspace
+      - name: Cleanup trivy cache
         run: |
-          sudo rm -rf /home/runner/work/wave/wave/build-workspace
+          sudo rm -rf /home/runner/work/wave/wave/build-workspace/.trivy-cache
```

Comment:
Is this problem still around? why only .trivy-cache ?

Reply:
yes, the problem is still around, I think in the previous command its not delete the hidden folders, and if i use .*, it does not execute, the problem is related to the access of `.trivy-cache`. So i tried it in this PR https://github.com/seqeralabs/wave/pull/503 and ci start working 
<img width="1002" alt="Screenshot 2024-05-23 at 15 54 11" src="https://github.com/seqeralabs/wave/assets/6575288/305c5f78-1f0f-4baa-8564-ec955a0ba4f7">

Reply:
previous tries

1. with `sudo rm -rf /home/runner/work/wave/wave/build-workspace/.*`
<img width="877" alt="Screenshot 2024-05-23 at 15 56 57" src="https://github.com/seqeralabs/wave/assets/6575288/f9a059be-6167-4209-a502-f9ee08568f35">

3. with `sudo rm -rf /home/runner/work/wave/wave`
<img width="1462" alt="Screenshot 2024-05-23 at 15 57 19" src="https://github.com/seqeralabs/wave/assets/6575288/bddeee0c-6b28-4785-b379-0db1e9ec4cbc">

Reply:
I'm quite sure `rm -rf /foo/bar` remove ALL bar content and the directory itself. therefore it would solve

Reply:
you are right, the problem is github ci skip this delete step in some failures, i am not sure why 
as happened in this run
https://github.com/seqeralabs/wave/actions/runs/9206439473/job/25324299024

Reply:
Likely [this](https://github.com/seqeralabs/wave/commit/8adb52ab355dd72ff3d4d19a51d42a63fdc2e546) should solve